aikau/src/main/resources/alfresco/core/topics.js

```diff
@@ -652,6 +652,21 @@ define([],function() {
       GET_PREFERENCE: "ALF_PREFERENCE_GET",
 
       /**
+       * This topic can be published to request a list of all available users in the Alfresco Repository.
+       * This request is expected to be handled by the [UserService]{@link module:alfresco/services/UserService}.
+       * The results are published on 'alfResponseTopic' provided in the attribute "items".
```

Comment:
This block all made sense apart from the final 5 words. Can you explain better? Where is this attribute found?